docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -475,8 +475,8 @@ Do not store the values globally. Every time the value is needed call `ConfigRes
 
 #### AbstractController
 
-The `eZ\Bundle\EzPublishCoreBundle\Controller` now extends `Symfony\Bundle\FrameworkBundle\Controller\AbstractController` instead of `Symfony\Bundle\FrameworkBundle\Controller\Controller` which has limited access to the [service container](https://doc.ibexa.co/en/latest/guide/service_container).
-For details, see [Service Subscribers Locators](https://symfony.com/doc/5.0/service_container/service_subscribers_locators.html).
+The `eZ\Bundle\EzPublishCoreBundle\Controller` now extends `Symfony\Bundle\FrameworkBundle\Controller\AbstractController` instead of `Symfony\Bundle\FrameworkBundle\Controller\Controller` which has limited access to the [service container](https://doc.ibexa.co/en/latest/guide/api/public_php_api.md#service-container).
```

Comment:
In Ibexa 4.x it is now:
```suggestion
The `Ibexa\Bundle\Core\Controller` now extends `Symfony\Bundle\FrameworkBundle\Controller\AbstractController` instead of `Symfony\Bundle\FrameworkBundle\Controller\Controller` which has limited access to the [service container](https://doc.ibexa.co/en/latest/guide/api/public_php_api.md#service-container).
```